api/client-server/administrative_contact.yaml

```
@@ -308,12 +419,9 @@ paths:
         already associated with an account on this homeserver. This API should
         be used to request validation tokens when adding an email address to an
         account. This API's parameters and response are identical to that of
-        the |/register/email/requestToken|_ endpoint. The homeserver has the
-        choice of validating the email address itself, or proxying the request
-        to the ``/validate/email/requestToken`` Identity Service API as
-        identified by ``id_server``. It is imperative that the
-        homeserver keep a list of trusted Identity Servers and only proxies to
-        those that it trusts.
+        the |/register/email/requestToken|_ endpoint. The homeserver should validate
+        the email itself, either by sending a validation email itself or by using
+        a service it has control over.
```

```suggestion
        a service it trusts.
```

? (and elswhere) given that people can use vector.im for validation?

---

The MSC specifically says "control" and explicitly discourages trust.